test/integration_test.go

```diff
@@ -522,44 +522,6 @@ func TestTerragruntWorksWithLocalTerraformVersion(t *testing.T) {
 	validateDynamoDBTableExistsAndIsTagged(t, TERRAFORM_REMOTE_STATE_S3_REGION, lockTableName, expectedDynamoDBTableTags)
 }
 
-// Regression test to ensure that skip flags associated with the S3 bucket autocreation feature. At one point,
-// terragrunt was stripping the flags in a way that it was no longer being honored.
-// This bug only happens if terragrunt calls remotestate.NeedsInitialization on the s3 initializer, so we need to setup
-// a situation where terragrunt calls this. To do this, we first setup the module with local backend (so that
-// moduleNeedsInit is false), before switching to the remote backend.
-func TestTerragruntHonorsS3RemoteStateSkipFlagsRegression(t *testing.T) {
```

Comment:
This test runs Terragrunt first with a local backend and then with an S3 backend. Unfortunately, with TF 0.15, when you switch to the S3 backend, it shows an interactive prompt asking if you want to migrate state over from the local backend. This interactive prompt now causes the test to fail. I tried switching the second Terragrunt run from `validate` to `apply -input=false`, but the interactive prompt still came up.

Not sure how this worked before 0.15, but as per @yorinasub17's [comment here](https://github.com/gruntwork-io/terragrunt/pull/1399#issuecomment-714774114), this test doesn't quite test what we are looking for anyway, and give the complexity of working around an interactive prompt, I opted to remove it.